meta/contributor-handbook.md

```diff
@@ -0,0 +1,88 @@
+# Contributor Handbook
+
+This document lays out a basic guideline of how to write and contribute a new pattern.
+
+## Maturity Levels
+
+We defined four maturity levels for patterns: The higher a pattern's maturity level, the more strictly it was validated and the better is its writing.
+
+### In a Nutshell
+
+| \# | Name | Description | Usage | Time to merge |
+| ---- | ---- | ---- | ---- | ---- |
+| 1 | Initial | Contains an initial pattern idea or what we call a donut (a pattern with missing sections). | Not included in any publications | A few days / weeks |
+| 2 | Structured | Contains a complete pattern that it is not properly validated yet (e.g. because it only synthesizes an idea or the experiences from one instead of three organizations). | Included in the patterns catalogue but with a warning | A few days / weeks |
+| 3 | Validated | The pattern is validated (e.g. because three or more instances exists and are synthesized by the pattern) and its writing is of high quality. | Included in the patterns catalogue | One or two months |
+| 4 | Holy | The pattern is validated, its writing is of "text book" quality, and the community agrees it is responsible to use it as teaching material. | Included in the patterns catalogue and patterns book | As long as it takes |
+
+For the first pattern you contribute, you should aim for maturity levels `1: initial` or `2: structured`. 
+
+If you want to help promoting a pattern one maturity level up, we suggest to first create an issue to discuss the matter and see if someone else is working on that already. After that, you can create a pull request with the necessary changes.
+
+To achieve a given maturity level a pattern has to satisfy the requirements for that given maturity level and lower levels. The following sections lay out the requirements per maturity level in detail.
+
+
+### Requirements: Level 1 - Initial
+
+Patterns (or other documents) of level 1 are stored in the directory `/1-initial`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	- Donuts - patterns without solution or other missing sections (`donut-*.md`)
+	- Tale - unstructured information in any form (`tale-*.md`)
+	
+- Validation requirements:
+	- N/A
+	
+- Content requirements:
+	- The document is readible & comprehensible for other's than the author(s)
+	- The author(s) contribute the contents according to the [license](../LICENSE.txt) & are allowed to do so
+	- Thoughts and contents by third parties are quoted / referenced explicitly
+
+	
+### Requirements: Level 2 - Structured
+
+Patterns of level 2 are stored in the directory `/2-structured`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	
+- Validation requirements:
+	- N/A

```

Comment:
I think it would be helpful to note that this pattern has been used in at least one instance (at least a partial validation).

Reply:
Thanks for this feedback @NewMexicoKid and @fwan2000 :) Would the following work for you?
```suggestion
	- Is partially validated at least one known instance
		- Or: pattern warrants to still be included (e.g. because validation is not feasible in the foreseeable future
```

My reasoning behind that loophole: I expect that for a few (less than a handfull) future patterns it might not be possible to provide validation validated within a time frame of maybe years. Still, we want to open up a path to let them mature (after a careful discussion in the community). This should be an extraordinary case.

Reply:
Thanks for clarifying @maxcapraro . There's need/value to capture ideas/proposals as patterns for others to implement/validate. While proven patterns are good for adoption, there're times people are looking for ideas to tackle their problem even though it's not been tried before. 

I guess for people looking for ideas, they can look across structured, proven & established patterns knowing they have different maturity level. For people looking for proven patterns or teaching, that would be restricted to proven & established patterns? If that's the case, structured can mean reviewed and approved by the community even though it's not validated. That sounds reasonable to me.